circle.yml

```diff
@@ -0,0 +1,13 @@
+machine:
+  node:
+    version: 4
+
+dependencies:
+  pre:
+    - nvm install 7
+
+test:
+  override:
+    - npm test
+    - nvm use 6 && npm test
+    - nvm use 7 && npm test
```

Comment:
I can add 8 additionally. These tests run pretty fast and we'll be sure it's actually bulletproof.